djaoapp/static/js/djaodjin-metrics.js

```diff
@@ -70,6 +70,8 @@ function updateChart(container, data, unit, dataScale, extra) {
     } else {
         defaults["currency"] = [unit, ""];
     }
+    var chartMobileWidth = 600;
+    var mobileBreakpoint = 576;
```

Comment:
These variables need to be derived from `$grid-breakpoints` in `scss/vendor/bootstrap/_variables.scss`.

Reply:
how?